common-functions.js

```diff
@@ -74,4 +74,167 @@ function monthNameWiseFormate(b) {
 function dateFormat(c) {
 	var days=['','1st','2nd','3rd','4th','5th','6th','7th','8th','9th','10th','11th','12th','13th','14th','15th','16th','17th','18th','19th','20th','21st','22nd','23rd','24th','25th','26th','27th','28th','29th','30th','31st']
 	return days[c];
-}
\ No newline at end of file
+}
+
+
+function checkArrayEmptyOrNot(array) {
+	if (array === 'undefined' || array.length == 0||(array[0]=="")) return true;
+	else return false;}
+
+	function deleteArrayRec(array, delIndex) {
+		arr=[];
+		for(var i=0;i<array.length;i++){
+			if(array[i]==delIndex) continue;
+			else arr.push(array[i]);
+		}
+		return arr;
+	}
+
+	function deleteMultipleRecFromArray(array, delIndex) {
+		for(var i=0; i<array.length; i++){
+			for(var j=0;j<delIndex.length;j++){
```

Comment:
I don't think this function will work correctly, recheck again.
second add space after for, if, etc.